src/install/apache/whatsNext.mdx

```diff
@@ -0,0 +1,637 @@
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+
+  <Collapser
+    id="example5"
+    title="Metrics only with alternative certificate"
+  >
+    In this configuration we only have one integration block with `METRICS: true` to collect only metrics and added `CA_BUNDLE_FILE` pointing to an alternative certificate file:
+
+    ```yml
+    integrations:
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: https://my_apache_host/server-status?auto
+          CA_BUNDLE_FILE: '/etc/ssl/certs/custom-ca.crt'
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+
+  <Collapser
+    id="example6"
+    title="Environment variables replacement"
+  >
+    In this configuration we are using the environment variable `APACHE_STATUS` to populate the STATUS_URL setting of the integration:
+
+    ```yml
+    integrations:
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: { { APACHE_STATUS } }
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+
+  <Collapser
+    id="example7"
+    title="Multi-instance monitoring"
+  >
+    In this configuration we are monitoring multiple Apache servers from the same integration. For the first instance (`STATUS_URL: https://1st_apache_host/server-status?auto`) we are collecting metrics and inventory while for the second instance (`STATUS_URL: https://2nd_apache_host/server-status?auto`) we will only collect metrics.
+
+    ```yml
+    integrations:
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: https://1st_apache_host/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+      - name: nri-apache
+        env:
+          INVENTORY: "true"
+          STATUS_URL: https://1st_apache_host/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 60s
+        labels:
+          env: production
+          role: load_balancer
+        inventory_source: config/apache
+
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: http://2nd_apache_host/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+</CollapserGroup>
+
+### Labels [#labels]
+
+You can further decorate your metrics using labels. Labels allow you to add attributes (key/value pairs) to your metrics, which you can then use to query, filter, or group your metrics.
+
+Our default sample config file includes examples of labels but, because they're not mandatory, you can remove, modify, or add new ones of your choice.
+
+```yml
```

Comment:
Should we preface this description with what this particular label is signifying? Is it worth adding some additional examples of what customeers typically would add that isn't out of the box. e.g., are there certain key/value pairs customers are expecting that we don't offer? should we include some of those?

Reply:
Labels are hyper-specific to user use-cases, but I think adding more context to the example is good!